synthesizer/src/program/finalize/command/get_or_use.rs

```diff
@@ -81,7 +81,7 @@ impl<N: Network> GetOrInit<N> {
     }
 }
 
-impl<N: Network> GetOrInit<N> {
+impl<N: Network> GetOrUse<N> {
     /// Finalizes the command.
     #[inline]
     pub fn finalize<P: FinalizeStorage<N>>(
```

Comment:
You can change this function to return `Result<()>`.

In the parent function that calls this function, you can add `.map(|_| None)` to the end of the call.